src/App.tsx

```diff
@@ -1,18 +1,53 @@
 import React from 'react';
 
 import './App.scss';
+import { Tabs } from './components/Tabs';
 
 const tabs: Tab[] = [
   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
-const App: React.FC = () => (
-  <div className="App">
-    <h1>Selected tab is&nbsp;Tab 1</h1>
-    <div>{tabs.length}</div>
-  </div>
-);
+type State = {
+  selectedTab: Tab;
+  selectedTabId: string;
+};
+
+class App extends React.PureComponent<{}, State> {
+  state: State = {
+    selectedTab: tabs[0],
+    selectedTabId: 'tab-1',
```

Comment:
just save selectedTab in state, you can get id from it

```suggestion
    selectedTab: tabs[0],
```